native/src/main/java/io/ballerina/stdlib/oracledb/utils/Utils.java

```diff
@@ -106,51 +102,6 @@ public static Properties generatePoolProperties(BMap clientOptions) {
         return null;
     }
 
-    private static void setSSLConProperties(BMap secureSocket, Properties connProperties) {
```

Comment:
We cannot remove functionality just to increase the code coverage. Let's leave the functionality as it is and send another PR is needed to cover this part.

Reply:
As @Anupama akka mentioned, first task is to increase the 80% of the coverage of the oracledb module and then keep on adding functionality with test cases. But to add ssl test cases will take more time as there must be another docker container to test ssl connections and we have not finalized the docker containers count to be used when running test cases. 

I have create a another issue https://github.com/ballerina-platform/ballerina-standard-library/issues/1672 to track the SSL code and will add the functionality in near future with proper test cases.

Reply:
The intention is to increase the code coverage with the current functionalities as soon as possible before working on adding new functionalities. Here the SSL connection functionality is already available. It is not a new feature to remove.

Reply:
Agreed. we can't remove the existing functionalities. It's okay to add test cases in a separate PR, but let's not remove the existing ones.

Reply:
added back in https://github.com/ballerina-platform/module-ballerinax-oracledb/pull/7/commits/903f0cd0b25dea933b9106c0947ee5a969e13afd